exatomic/gaussian/output.py

```diff
@@ -499,11 +507,13 @@ def parse_frequency(self):
             # Get the frequencies first
             freqs = ln[freq_start:].split()
             # get the ir intensities
-            ir_int = self[lno+3][freq_start:].split()
+            ir_int = list(map(float, self[lno+3][freq_start:].split()))
+            r_mass = list(map(float, self[lno+1][freq_start:].split()))
             nfreqs = len(freqs)
             # Get just the atom displacement vectors
             start = lno + span_freq_vals
             stop = start + span
+            #print(lno, start, stop, self[start], self[stop])
```

Comment:
I would delete debug print statements. Likewise throughout.

Reply:
Made a new commit addressing this.